lib/rules/no-named-import.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @fileoverview No named imports on React
+ * @author Angelo Abdoelsamath
+ */
+
+'use strict';
+
+const docsUrl = require('../util/docsUrl');
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'No named imports on React',
+      category: 'Best Practices',
+      recommended: false,
+      url: docsUrl('no-named-import')
+    },
+    schema: [
+      {
+        oneOf: [
+          {
+            type: 'object',
+            properties: {
+              allow: {
+                type: 'array'
+              },
+              forceImport: {
+                type: 'boolean'
+              }
+            },
+            additionalProperties: false
+          },
+          {
+            type: 'object',
+            properties: {
+              forbid: {
+                type: 'array'
+              },
+              forceImport: {
+                type: 'boolean'
+              }
+            },
+            additionalProperties: false
+          }
+        ]
+      }
+    ],
+    messages: {
+      dontImport: "Don't import {{name}} from React. Use React.{{name}} to be consistent.",
+      shouldImport: 'Import {{name}} from React'
+    }
+  },
+  create(context) {
+    // ignore non-modules
+    if (context.parserOptions.sourceType !== 'module') {
+      return {};
+    }
+    const {options} = context;
```

Comment:
Destructuring syntax is not supported in Node 4. For debugging without istanbul messing up codes you can run `$(npm bin)/mocha tests/lib/rules/no-named-import.js`.

Reply:
Thanks! It's fixed :)